### ISSUE TITLE
Value field expanded

### ISSUE THREAD

**Issue by marcozanca** [thomasnordquist/MQTT-Explorer#522]
First of all: great work. I find this to be the nicest mqtt viewer out there. 

It would be nice if the value field could use all the space available. For example in this project I have some long json and I have to scroll every time even if I collapsed the Publish, Stats and Topic fields. 

![immagine](https://user-images.githubusercontent.com/958611/110599960-b4443580-8183-11eb-93d6-2d0a7b6ec6a6.png)

Thanks

**Comment by benchdollar** [comment]
Yes, this is the no #1 feature I wish for every time I use this awesome software!
